docs/proposals/22-autoscaling-kube-apiserver-via-independent-hpa-and-vpa.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Autoscaling Shoot `kube-apiserver` via Independently Driven HPA and VPA  
+gep-number: 0022
+creation-date: 2023-01-31
+status: implementable
+authors:
+- "@andrerun"
+- "@voelzmo"
+reviewers:
```

Comment:
Who will be the main reviewers for this GEP?

Reply:
Added oliver-goetz as main reviewer.